modules/javafx.controls/src/main/java/com/sun/javafx/scene/control/behavior/ListViewBehavior.java

```diff
@@ -138,7 +140,8 @@ public ListViewBehavior(ListView<T> control) {
             new KeyMapping(new KeyBinding(PAGE_DOWN).shortcut().shift(), e -> discontinuousSelectPageDown())
         );
 
-        if (!Boolean.TRUE.equals(control.getProperties().containsKey("excludeKeyMappingsForComboBoxEditor"))) {
+        if (Boolean.FALSE.equals(control.getProperties().containsKey("excludeKeyMappingsForComboBoxEditor"))) {
```

Comment:
Unless I'm missing something, you don't need to compare with a `Boolean` at all since containsKey returns a `boolean` primitive type, so I think this can be simplified to:

```
if (!control.getProperties().containsKey("excludeKeyMappingsForComboBoxEditor")) {
```

If instead you do want to check the value of the property, and not just its existence, you would need the following, and it would be important to check `!TRUE.equals` rather than `FALSE.equals` so that an unset property would be treated as `false`.

```
if (!Boolean.TRUE.equals(control.getProperties().get("excludeKeyMappingsForComboBoxEditor"))) {
```

Reply:
Changed to use
`if (!control.getProperties().containsKey("excludeKeyMappingsForComboBoxEditor")) {`